starlette_admin/templates/login.html

```diff
@@ -33,7 +33,7 @@ <h2 class="card-title text-center mb-4">{{ _("Login to your account") }}</h2>
                     <input type="text"
                            name="username"
                            class="form-control {% if form_errors and form_errors.has('username') %}is-invalid{% endif %}"
-                           placeholder="Enter username"
+                           placeholder="{{ _('Enter username') }}"
```

Comment:
```suggestion
                           placeholder="{{ _('Username') }}"
```
Let's make it general